findoutlie/detectors.py

```diff
@@ -46,8 +48,16 @@ def iqr_detector(measures, iqr_proportion=1.5):
         corresponding value in `measures` is an outlier.
     """
     # Any imports you need
+    
     # Hints:
     # * investigate np.percentile
     # * You'll likely need np.logical_or
     # https://textbook.nipraxis.org/numpy_logical.html
     # +++your code here+++
+
+    #Calculating the quartiels and the IQR for some vector called 'measures'
+    Q1 = np.percentile(measures,25)
+    Q3 = np.percentile(measures,75)
+    IQR = Q3 - Q1
+    outlier_tf = np.logical_or(measures < (iqr_proportion*IQR),measures > (iqr_proportion*IQR))
```

Comment:
I made a mistake this should read  ... (measures < (Q1-(iqr_proportion*IQR)),measures > (Q3+(iqr_proportion*IQR)))